doc/generic/pgf/text-en/pgfmanual-en-base-animations.tex

```diff
@@ -880,11 +880,11 @@ \subsection{Animating Paths and their Rendering}
     path consists of appropriate Bézier curves.
 
     Unlike the dash pattern, the to-be-animated object is, indeed, the path
-    itself and not some special scope. This means that you can use the 
-    |current value| for the start path. However, this also means that you really must
-    pick \emph{the path object} as the target of the animation. In conjunction
-    with \tikzname, this will be an object of type |path| as in the above
-    example.
+    itself and not some special scope. This means that you can use the
+    |current value| for the start path. However, this also means that you
+    really must pick \emph{the path object} as the target of the animation. In
+    conjunction with \tikzname, this will be an object of type |path| as in the
+    above example.
```

Comment:
Stop changing the spacing!  You didn't even touch this paragraph.  Just learn to configure your stupid editor!